internal/hooks/hooks.go

```
@@ -152,24 +152,111 @@ const maxNameLength = 80
+	return WriteWhole(m.NameResultFile(id), content)
+}
+
+// ReadNameResult returns the name that was asked for and what became of
+// it: the applied name, or the refusal as an error. found reports that
+// the poller has answered, so a caller compares requested against its own
+// to know whether the answer is the one it is waiting for. Anything but
+// the two verdicts this package writes is no answer at all.
+func (m *Manager) ReadNameResult(id string) (requested, applied string, refusal error, found bool) {
+	raw, err := os.ReadFile(m.NameResultFile(id))
+	if err != nil {
+		return "", "", nil, false
```

golangci-lint (2.13.2)

[error] error is not nil (line 238) but it returns nil

(nilerr)